src/app/components/MessageBanner/index.tsx

```diff
@@ -0,0 +1,82 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import React, { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import Paragraph from '../Paragraph';
+import Heading from '../Heading';
+import Image from '../Image';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import styles from './index.styles';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+interface MessageBanner {
+  summaries: {
+    imageUrl?: string;
+    link: string;
+    imageAlt?: string;
+    description?: string;
+    title: string;
+    id: string;
+  }[];
+  title: string;
+  position: number;
+}
+const MessageBanner = ({ summaries, title, position = 0 }: MessageBanner) => {
+  const { dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  return (
+    <section
+      css={styles.container}
+      role="region"
+      aria-labelledby={`message-banner-${position}`}
+    >
+      <div css={styles.card}>
+        {summaries.map(summary => {
```

Comment:
We have created a ticket - **WSTEAMA-220- Refactor the structure of the message banner component to only accept heading, description, link and image** - to refactor this in a future iteration - we don't think it makes sense to map over summaries here, but rather just have the heading, description, link, image (and perhaps a unique identifier)